File: test/acceptance/pages/UserPage.java
package acceptance.pages;

import acceptance.SharedDriver;
import org.openqa.selenium.By;

public class UserPage {
  private final SharedDriver driver;

  public UserPage(SharedDriver driver) {
    this.driver = driver;
  }

  public boolean isViewing(String userName) {
    return driver.findElement(By.id("page_name")).getText().startsWith(userName);
  }
}

File: test/se/crisp/signup4/web/pages/UserPage.scala
package se.crisp.signup4.web.pages

import org.openqa.selenium.{By, WebDriver}

class UserPage(baseUrl: String, driver:WebDriver) extends Page(baseUrl, driver) {
  def isViewing(userName: String): Boolean = driver.findElement(By.id("page_name")).getText.startsWith(userName)
}
